scaletest/templates/scaletest-runner/scripts/lib.sh

```diff
@@ -183,11 +183,20 @@ annotate_grafana_end() {
 
 	log "Annotating Grafana (end=${end}): ${text} [${tags}]"
 
-	json="$(
-		jq \
-			--argjson timeEnd "${end}" \
-			'{timeEnd: $timeEnd}' <<<'{}'
-	)"
+	if [[ -n ${GRAFANA_ADD_TAGS:-} ]]; then
+		json="$(
+			jq \
+				--argjson timeEnd "${end}" \
+				--argjson tags "${tags},${GRAFANA_ADD_TAGS}" \
+				'{timeEnd: $timeEnd, tags: $tags | split(",")}' <<<'{}'
```

Comment:
I think you can also do `jq -n` instead of reading an empty object?

Reply:
Thanks, I'll try that!